docs/06_how-to-guides/40_multi-index/how-to-instantiate-a-multi-index-table.md

```diff
@@ -1,32 +1,65 @@
 ---
-content_title: How to instantiate a multi index table
+content_title: How to instantiate a multi-index table
 ---
 
-1. Include the `eosio.hpp` header and declare the `eosio` namespace usage
-```
+## Overview
+
+This guide provides instructions to instantiate a multi-index table.
+
+## Reference
+
+See the following code reference:
+
+* The [`multi-index`](../../classeosio_1_1multi__index) class.
+
+## Before you begin
+
+Make sure you have the following prerequisites in place:
+
+* An EOSIO development environment, for details consult the [Get Started Guide](https://developers.eos.io/welcome/latest/getting-started-guide/index),
+
+## Procedure
+
+Complete the following steps to instantiate a multi-index table `testtab`.
+
+### 1. Preparation And Initialization
+
+Include the `eosio.hpp` header and use the `using` directive to access the `eosio` namespace.
+
+```cpp
 #include <eosio/eosio.hpp>
 using namespace eosio;
 ```
-2. Define the data structure for the multi index table
+
+### 2. Define The Table Data Structure
+
+Define the data structure for the multi-index table.
+
 ```cpp
   struct [[eosio::table]] test_table {
   };
 ```
-3. Add to the data structure the fields which define the multi index table
-```diff
-  // the data structure which defines each row of the table
-  struct [[eosio::table]] test_table {
-+    // this field stores a name for each row of the multi index table
-+    name test_primary;
-+    // additional data stored in table row, e.g. an uint64_t type data
-+    uint64_t datum;
-  };
-```
-4. Add definition of the primary index for the multi index table. The primary index type must be uint64_t, it must be unique and it must be named `primary_key()`, if you don't have this the compiler (eosio-cpp) will generate an error saying it can't find the field to use as the primary key:
+
+Add to the data structure the properties which define it. Each data member corresponds to a field of the multi-index table. A primary key is required when defining a multi-index table structure, therefore you need to know which is the multi-index table field that is the primary key for your multi-index table. The corresponding data member for the primary key field must store unique values. In this case it is the `test_primary` data member of type `eosio::name`.
```

Comment:
this should be the same where ever this paragraph is used - see above:

Add the data structure properties(data members???) . Each data member corresponds to a field of the multi-index table. A primary key is required when defining a multi-index table structure,